### PR TITLE
[ENHANCEMENT] - Clearing the configured app property when there are no connections

### PR DESCRIPTION
**Pull request by krazziekay**
**What's in this PR?**
- Set the property to false when there are no connections.

### CODE REVIEW

File: src/util/is-connected.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Subscription } from "models/subscription";
+import { GitHubServerApp } from "models/github-server-app";
+import { Request } from "express";
+import { getConnectionsAndInstallations } from "routes/jira/jira-get";
+
+
+export const isConnected = async (req: Request, jiraHost: string, installationId: number): Promise<boolean> => {
+	const subscriptions = await Subscription.getAllForHost(jiraHost) || [];
```
Comment (gxueatlassian) [inline src/util/is-connected.ts:8]: Is there a reason we can't just do `return subscriptions.length >0`?

**Review by atrigueiro — APPROVED** [review]

File: src/util/is-connected.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	const gheServers: GitHubServerApp[] = await GitHubServerApp.findForInstallationId(installationId) || [];
```
Comment (gxueatlassian) [inline src/util/is-connected.ts:9]: Tbh, I don't think `GitHubServerApp.findForInstallationId(installationId)` is safe, as the `installationId` can be overlap between different customers

**Review by gxueatlassian — CHANGES_REQUESTED** [review]
Just some comments.

File: src/util/is-connected.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Subscription } from "models/subscription";
+import { GitHubServerApp } from "models/github-server-app";
+import { Request } from "express";
+import { getConnectionsAndInstallations } from "routes/jira/jira-get";
```
Comment (gxueatlassian) [inline src/util/is-connected.ts:4]: This is a bit strange in terms of importing. `jira-delete` --> import `utils/is-connecrted` --> import `jira-get`.

File: src/util/is-connected.ts (hunk 1 above)
Comment (krazziekay) [reply src/util/is-connected.ts:8]: That will only work if there were cloud apps. The enterprise server works a bit differently. Don't exactly remember the relation right now but I've been testing and it seems that the GitHub server apps that have been deleted will only delete the entry in the table `GitHubServerApps` and their corresponding `Subscription` is not deleted.

Comment (krazziekay) [reply src/util/is-connected.ts:8]: Ok, when we're deleting a GitHub server app, its corresponding subscription is not deleted. It just stays there for now. That needs to be done first if we wanna use `subscriptions.length`. 
Adding a TODO for that now and have created a ticket for that. That needs to be tackled separately.

**Review by gxueatlassian — APPROVED** [review]

**Pull request merged**